tests/src/utils/mt.rs

```diff
@@ -48,6 +48,21 @@ pub trait MtExt {
         account_id: &AccountId,
         token_ids: impl IntoIterator<Item = &TokenId>,
     ) -> anyhow::Result<Vec<u128>>;
+
+    async fn mt_tokens(
+        &self,
+        token_contract: &AccountId,
+        from_index: Option<U128>,
```

Comment:
Can you make it accept `impl RangeBounds<usize>`? It will be more convenient to call it if the conversion happens in impl of this trait